massgen/tests/test_coordination_config_wiring.py

```
@@ -0,0 +1,153 @@
+                "decomposition_guidelines": "Prefer independent chunks.",
+                "timeout_seconds": 120,
+            },
+            "subagent_orchestrator": {
+                "enabled": True,
+                "inherit_spawning_agent_backend": True,
+                "shared_child_team_types": ["round_evaluator", "builder"],
+                "final_answer_strategy": "synthesize",
+            },
+            "standalone_checkpoint": {
+                "enabled": True,
+                "team_config": "/tmp/team.yaml",
```

Ruff (0.15.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/team.yaml"

(S108)